public/scripts/backupSheets.js

```diff
@@ -49,39 +43,57 @@ function backupSheets() {
       sheet_id: "1GwP7Ly6iaqgcms0T80QGCNW4y2gJ7tzVND2CktFqnXM",
       backup_folder_id: "1gaKQLiUxeIHGJl3it-sDpAtjGEAhT-wg"
     }];
-  
+
+const SHEETS_FOLDER = "1scY72enARHyBS5nKow6qeT4TLH9feL09";
+
+/*
+  Create a backup of each country's sheet in a country-specific
+  Google Drive folder.
+*/
+function backupSheets() {
+    
   const formattedDate = Utilities.formatDate(new Date(), "PST", "yyyy-MM-dd' 'HH:mm:ss");
   
-  const sheetsFolder = DriveApp.getFolderById("1scY72enARHyBS5nKow6qeT4TLH9feL09");
+  const sheetsFolder = DriveApp.getFolderById(SHEETS_FOLDER);
   
-  for (let i = 0; i < sheets.length; i++) {
-    const name = "Backup of " + sheets[i].name + " " + formattedDate;
-  
-    const file = DriveApp.getFileById(sheets[i].sheet_id);
-  
-    const destination = DriveApp.getFolderById(sheets[i].backup_folder_id);
-  
-    // Makes a copy of file with name "name" in the "destination" folder.
-    file.makeCopy(name, destination);
+  for (let i = 0; i < SHEETS.length; i++) {
```

Comment:
nit;

for (const sheet of SHEETS) {
  const name = "Backup of " + sheet.name + " " + formattedDate;
  ....
}

Reply:
Ooooooh.  Very cool.  Thank you, sensei. 🙇‍♀